k8s/oidc-aws/server-statefulset.yaml

```diff
@@ -69,11 +69,6 @@ spec:
           - name: spire-data
             mountPath: /run/spire/data
             readOnly: false
-          readinessProbe:
```

Comment:
What do you think about doing an HTTP probe to the "/keys" endpoint instead? Will that work? That seems better than nothing.

Reply:
That's a good suggestion. I updated the probes to use `/keys` for now.